schema/patches/13.sql

```diff
@@ -0,0 +1,26 @@
+create table uryplayer.podcast_category
+(
+	podcast_category_id serial not null,
+	category_name text not null
+);
+
+create unique index podcast_category_category_name_uindex
+	on uryplayer.podcast_category (category_name);
+
+create unique index podcast_category_podcast_category_id_uindex
+	on uryplayer.podcast_category (podcast_category_id);
+
+alter table uryplayer.podcast_category
+	add constraint podcast_category_pk
+		primary key (podcast_category_id);
```

Comment:
This could be done in the `create table` (`podcast_category_id SERIAL NOT NULL PRIMARY KEY`